### PR TITLE
Json server

### PR DESCRIPTION
**Pull request by myrosvas**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "author": "andriytereschuk",
   "private": true,
   "scripts": {
+    "all": "nuxt & json-server --watch db.json --port 3001",
     "dev": "nuxt",
```
Comment (andriytereschuk) [inline package.json]: let use existing script
`"dev": "nuxt & json-server --watch db.json --port 3001"`

File: store/project.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import EventService from './../services/ProjectService'
```
Comment (andriytereschuk) [inline store/project.js]: EventService?

File: pages/projects/index.vue [hunk 3]
```diff
@@ -41,14 +38,24 @@
...
-const { mapGetters, mapActions, mapMutations } = createNamespacedHelpers(
-  'projects'
-)
+const { mapGetters, mapActions } = createNamespacedHelpers('projects')
 
 export default {
   data: () => {
     return {
       dialog: false,
+      newProject: {
```
Comment (andriytereschuk) [inline pages/projects/index.vue]: supposed to be in the `project` store, isn't?

File: pages/projects/index.vue [hunk 4]
```diff
@@ -88,11 +95,16 @@ export default {
   },
   methods: {
     ...mapActions({
-      fetch: 'get'
-    }),
-    ...mapMutations({
-      remove: 'remove'
+      fetch: 'get',
+      addProject: 'addProject',
+      deleteProject: 'deleteProject'
```
Comment (andriytereschuk) [inline pages/projects/index.vue]: it could be duplication.. since we have `project` store.
check please

Comment (andriytereschuk) [reply pages/projects/index.vue]: check whether we can delegate to `project` store

File: pages/projects/index.vue (hunk 3 above)
Comment (myrosvas) [reply pages/projects/index.vue]: The logic here looks like this: when user clicks on New Project button, the new project is created and is added to the `projects` array in the db.json and in the application state. As the `projects` array is being modified, I placed the functions in the store/projects.

**Pull request merged**